pkg/sidecar/docker_reactor.go

```diff
@@ -122,6 +123,12 @@ func (d *DockerReactor) handleContainer(ctx context.Context, container *docker.C
 		return nil, nil
 	}
 
+	if strings.Contains(info.Name, "mkdir-outputs") {
```

Comment:
Since `containers` in `pods` share the env variables that we set, currently we try to process both containers in the testplan pod - `mkdir-outputs` and the actual testplan container.

If the `mkdir-outputs` doesn't exit right away and sidecar detects it, sometimes we even go as far as trying to add the `weave` network to it, before the context is canceled and the operation is stopped.

Reply:
slightly offtopic, but - now that `testground daemon` is on k8s, we could very well mount EFS on it, and have the cluster_k8s.go runner init the output folders prior to a testplan run.